app/components/course-page/base-stages-completed-card.ts

```
@@ -1,7 +1,12 @@
-import { tracked } from '@glimmer/tracking';
 import Component from '@glimmer/component';
-import congratulationsImage from '/assets/images/icons/congratulations.png';
 import RepositoryModel from 'codecrafters-frontend/models/repository';
+import congratulationsImage from '/assets/images/icons/congratulations.png';
+import type CoursePageStateService from 'codecrafters-frontend/services/course-page-state';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import type RouterService from '@ember/routing/router-service';
+import { waitFor } from '@ember/test-waiters';
```

ESLint

[error] 'waitFor' is defined but never used.

(@typescript-eslint/no-unused-vars)